code/obj/critter/misc.dm

```diff
@@ -487,22 +489,42 @@
 			else
 				continue
 
+	attackby(obj/item/W as obj, mob/M as mob)
+		if(istype(W, /obj/item/reagent_containers/food/snacks) && !(M in src.friends))
+			if(prob(20))
+				src.visible_message("<span class='notice'>[src] chitters happily at the [W], it will no longer attack [M]!</span>")
```

Comment:
Since messages are a little less narration and a little more textual clue-ins to what's going on ingame, this seems like a better way to go about it

```suggestion
				src.visible_message("<span class='notice'>[src] chitters happily at the [W], and seems a little friendlier with [M]!</span>")
```